File: iftest.cpp
#include <iostream>

using namespace std;

int main(int argc, char const *argv[])
{
 /* code */
 int nombre1 = 0;
 int nombre2 = 0;

 cout << "Entre un premier nombre: ";
 cin >> nombre1;
 cout << nombre1 << endl;

 cout << "Entre un deuxième nombre: ";
 cin >> nombre2;
 cout << nombre2;

 if (nombre1 > nombre2)
 {
  cout << nombre1 << "est plus grad que" << nombre2 << endl;
 }
 else
 {
  cout << nombre1 << "est plus petit que" << nombre2 << endl;
 }

 return 0;
}

File: IfTest.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class IfTest {
 public static void main(String[] args) {
  try {
   int nombre1 = 0;
   int nombre2 = 0;

   BufferedReader stdin = new BufferedReader(new InputStreamReader(System.in));
   System.out.print("Ente un premier nombre: ");
   nombre1 = Integer.parseInt(stdin.readLine());
   System.out.println(nombre1);
   System.out.print("Entre un second nombre: ");
   nombre2 = Integer.parseInt(stdin.readLine());
   System.out.println(nombre2);
   if (nombre1 > nombre2) {
    System.out.print(nombre1 + " est plus grand que " + nombre2);
   } else {
    if (nombre1 < nombre2) {
     System.out.print(nombre1 + " est plus petit que " + nombre2);

    } else {
     System.out.print(nombre1 + " est égal à " + nombre2);
    }
   }

  } catch (IOException ioe) {

  }
 }

}
